framework/src/main/java/org/checkerframework/framework/type/treeannotator/PropagationTreeAnnotator.java

```diff
@@ -320,4 +363,49 @@ private void addAnnoOrBound(AnnotatedTypeMirror type, Set<? extends AnnotationMi
     type.addMissingAnnotations(annosToAdd);
     type.addMissingAnnotations(annos);
   }
+
+  /**
+   * Propagate annotations from the type arguments of initializer type to var.
+   *
+   * <p>Simple Example:
+   *
+   * <pre>{@code
+   * typeOf(initializerType) = ArrayList<@Untainted String>
```

Comment:
I don't understand this example.   `initializerType` is a type so why is it passed to `typeOf`?

Reply:
ah right. the `typeof` is redundant here.